### ISSUE TITLE
Curriculo1

### ISSUE THREAD

**Issue by geovannaque** [geovannaque/curiculo-geovanna#1]
<!DOCTYPE html>
<html lang="pt-BR">
<head>
  
  <title>Geovanna Alcântara</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  <style>
  body {
      font: 20px Montserrat, sans-serif;
      line-height: 1.8;
      color: #f5f6f7;
  }
  p {font-size: 16px;}
  .margin {margin-bottom: 45px;}
  .bg-1 { 
      background-color: #4378a8; 
      color: #ffffff;
  }
  .bg-2 { 
      background-color: #2f2f2f; /* Dark Blue */
      color: #ffffff;
  }
  .bg-3 { 
      background-color: #ffffff; /* White */
      color: #555555;
  }
  .bg-4 { 
      background-color: #474e5d; /* Black Gray */
      color: #fff;
  }
  .container-fluid {
      padding-top: 70px;
      padding-bottom: 70px;
  }
  .navbar {
      padding-top: 15px;
      padding-bottom: 15px;
      border: 0;
      border-radius: 0;
      margin-bottom: 0;
      font-size: 12px;
      letter-spacing: 5px;
  }
  .navbar-nav  li a:hover {
      color: #1abc9c !important;

  
  </style>
</head>
<body>

<div class="container-fluid bg-3 text-center">
  <br><br><img src="http://i.imgur.com/8VfewJQ.jpg" class="img-responsive margin" style="display:inline">
  <br><br><br><br>
 

<div class="container-fluid bg-1 text-center">
  <h3 class="margin">Quem sou eu?</h3>
  <img src="http://i.imgur.com/KVgCgUE.jpg" class="" style="display:inline" alt="Eu" width="350" height="350"><br><br>
  <h3>Geovanna Alcântara, 20 anos, graduanda de Sistemas de Informação pela UFVJM.</h3>
</div>

</div>

<div class="container-fluid bg-2 text-center">    
  <h3 class="margin">Curriculum</h3><br>
  <div class="row">
    <div class="col-sm-4">
      <p> 
      <strong>Geovanna Alcântara de Queiroz </strong> <br>
      Rua do Fogo, 361, apto K <br>
      Bom Jesus – Diamantina - MG <br>
      Telefone: (38) 992396400 <br>
      E-mail: geovannaaqueiroz@live.com<br></p>
    </div>
    <div class="col-sm-4"> 
      <p><strong>Formação </strong> <br>
      Ensino Superior em andamento <br>
      Bacharelado em Sistemas de Informação <br>
      Universidade Federal dos Vales do Jequitinhonha e Mucuri<br>
      Diamantina - MG <br>
</p>
      </div>
    <div class="col-sm-4"> 
      <p><strong>Experiência </strong> <br>
      NextStep Empresa Jr. de Sistemas de Informação<br>
      Gerente Jurídico, desde junho/2017 
      </p>
    </div>
  </div>
  <br><br>
</div>

<div class="container-fluid bg-3 text-center">
  <h3 class="margin">Aleatoriedades</h3> <br>
  <div class="col-sm-4"> 
      <p><strong>Meus filhos </strong> <br>
       <img src="http://i.imgur.com/dMcOACD.jpg" class="img-responsive margin" style="width:100%">
      
      </p>
   
    </div>
     <div class="col-sm-4"> 
      <p><strong>Meu Namorado </strong> <br>
       <img src="http://i.imgur.com/47g7ITH.jpg" class="img-responsive margin" style="width:100%">
      
      </p>
      
    </div>

  <div class="col-sm-4"> 
  <p><strong>Meu Tinder </strong> <br>
    <img src= "http://i.imgur.com/xynR6ML.jpg"class="img-responsive margin" style="width:100%">
</div>

</div>

<p>

<div class="container-fluid bg-3 text-center">  
<strong>Redes Sociais </strong> <br>
      <a href="https://www.linkedin.com/in/geovannaalcantara/"><img src="http://i.imgur.com/LkqSHN8.png"></a>
      <a href="https://www.facebook.com/geovannaaqueiroz"><img src="http://i.imgur.com/zVbkWEm.png"></a>
       <a href="https://www.instagram.com/geovannaque/"><img src="http://i.imgur.com/tEJu11V.png"></a>
      </p>
      </div>

<footer class="container-fluid bg-4 text-center">

  <p>Bootstrap Theme Made By <a href="https://www.w3schools.com">www.w3schools.com</a></p> 
</footer>

</body>
</html>
